apps/gitness/src/utils/__tests__/common-utils.test.ts

```diff
@@ -0,0 +1,151 @@
+import { describe, expect, it } from 'vitest'
```

Comment:
These are already set up as globals in the `vitest.config.ts` file
```suggestion
```

Reply:
Done, cleaned up from here.